### ISSUE TITLE
Error in read.graph.graphml in R 4.0.4

### ISSUE THREAD

**Issue by asar1245** [igraph/rigraph#457]
Hello everyone, I have an error reading graphml files. The error message is presented below:

Error in read.graph.graphml(file, ...) : 
  At foreign-graphml.c:599 : Cannot parse GraphML file, unknown attribute type, Parse error

I'm working in Debian 10 buster, and R version 4.0.4. 

I have installed the package libxml2-dev, but it does not help.

Thank you very much for your help.

**Comment by ntamas** [comment]
Please show us your GraphML file; it looks like it contains attribute types that are not handled by igraph (and might not even be in the GraphML specification).

**Comment by asar1245** [comment]
Sure, this is the compressed graphml file I'm trying to read: 
[antiparasitic_network_graph.graphml.zip](https://github.com/igraph/rigraph/files/6111652/antiparasitic_network_graph.graphml.zip)

Thank you very much in advance for your help.

**Comment by szhorvat** [comment]
I do not think that `string[]` is a valid attribute type, see http://graphml.graphdrawing.org/xmlns/1.1/graphml-attributes.xsd

**Comment by szhorvat** [comment]
It would be nice to report what the unknown attribute type is exactly (`string[]` in this case).

**Comment by szhorvat** [comment]
It would be even better to just issue a warning (mentioning the attribute name and type), but treat the unknown attribute as a string.
